optaplanner-examples/src/test/java/org/optaplanner/examples/common/experimental/ExperimentalConstraintCollectorsTest.java

```diff
@@ -123,8 +123,8 @@ void consecutiveInterval() {
 
     private ConsecutiveInfo<Integer, Integer> consecutiveData(Integer... data) {
         ConsecutiveSetTree<Integer, Integer, Integer> tree =
-                new ConsecutiveSetTree<>(Integer::intValue, (a, b) -> b - a, Integer::sum, 1, 0);
-        asList(data).forEach(tree::add);
+                new ConsecutiveSetTree<>((a, b) -> b - a, Integer::sum, 1, 0);
+        asList(data).forEach(datum -> tree.add(datum, datum)); // datum is the singular form of data
```

Comment:
```suggestion
        asList(data).forEach(datum -> tree.add(datum, datum));
```

Let's not use source code to educate people on Latin grammar :-)